linux/step01_debian9_pg_deps.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+PGVER=${PGVER:-pg96}
+
+# Postgres installation
+# 9.6
+#start-recommended
+apt-get -y install postgresql
```

Comment:
Ah, that's fine: it's unlikely to change until buster is released, I just wasn't sure if the "9.6" comment meant "this is probably going to be 9.6" or "we actually want 9.6", sounds like the former was the thinking. :+1: